docs/nlu/components.rst

```diff
@@ -114,40 +129,53 @@ SpacyFeaturizer
 :Short: spacy intent featurizer
 :Outputs: nothing, used as an input to intent classifiers that need intent features (e.g. ``SklearnIntentClassifier``)
 :Requires: :ref:`SpacyNLP`
+:Type: Dense featurizer
 :Description:
     Creates feature for intent classification using the spacy featurizer.
-    Optionally adds word vectors for each ``token`` to ``ner_features``, which can be referenced in ``CRFEntityExtractor``
 :Configuration:
 
     .. code-block:: yaml
 
         pipeline:
         - name: "SpacyFeaturizer"
-          # Whether to add word vectors to ``ner_features`` (default: False)
-          ner_feature_vectors: True
 
-NGramFeaturizer
-~~~~~~~~~~~~~~~
 
-:Short: Appends char-ngram features to feature vector
-:Outputs: nothing, appends its features to an existing feature vector generated by another intent featurizer
-:Requires: :ref:`SpacyNLP`
+ConveRTFeaturizer
+~~~~~~~~~~~~~~~~~
+
+:Short:
+    Creates a vector representation of user message and response (if specified) using
+    `ConveRT <https://github.com/PolyAI-LDN/polyai-models>`_ model.
+:Outputs:
+    nothing, used as an input to intent classifiers and response selectors that need intent features and response
+    features respectively (e.g. ``EmbeddingIntentClassifier`` and ``ResponseSelector``)
+:Requires: nothing
+:Type: Dense featurizer
 :Description:
-    This featurizer appends character ngram features to a feature vector. During training the component looks for the
-    most common character sequences (e.g. ``app`` or ``ing``). The added features represent a boolean flag if the
-    character sequence is present in the word sequence or not.
+    Creates features for intent classification and response selection.
+    Uses the `default signature <https://github.com/PolyAI-LDN/polyai-models#tfhub-signatures>`_ to compute vector
+    representations of input text.
 
-    .. note:: There needs to be another intent featurizer previous to this one in the pipeline!
+    .. warning::
+        Since ``ConveRT`` model is trained only on an english corpus of conversations, this featurizer should only
+        be used if your training data is in english language.
+
+    .. note::
+        To use ``ConveRTFeaturizer`` you should install ``tensorflow_text==1.15.1`` and ``tensorflow_hub==0.6.0``.
```

Comment:
the version numbers shouldn't be in here, they will get stale. I'd stick with the `rasa[convert]` suggestion and just mention which additional libs it installs.